test_unstructured_ingest/expected-structured-output/confluence/testteamsp/1867777.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "NarrativeText",
+    "element_id": "d2cfe2b37317a51a54267f553e9b1247",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/plain"
+    },
+    "text": "<p>Testdoc1 has only this text for 10 times: 1</p><p>Testdoc1 has only this text for 10 times: 2</p><p>Testdoc1 has only this text for 10 times: 3</p><p>Testdoc1 has only this text for 10 times: 4</p><p>Testdoc1 has only this text for 10 times: 5</p><p>Testdoc1 has only this text for 10 times: 6</p><p>Testdoc1 has only this text for 10 times: 7</p><p>Testdoc1 has only this text for 10 times: 8</p><p>Testdoc1 has only this text for 10 times: 9</p><p>Testdoc1 has only this text for 10 times: 10</p>"
+  }
+]
```

Comment:
any idea why some of these are coming through as text/plain rather than text/html?

Reply:
I'll check this to see why this happens

Reply:
Yes, `partition.auto` doesn't default to html parsing due to the file extension. Since it's safe to say [requests to confluence pages return valid html](https://github.com/Unstructured-IO/unstructured/pull/906#discussion_r1265222659), I'm changing file extensions to html as proposed.